### PR TITLE
Added get_md5_size_table, a batch script to obtain md5s and file size…

### PR DESCRIPTION
**Pull request by rahulg603**
…s for all files in a folder. Also fixed some bugs regarding importing h2 files.

### CODE REVIEW

File: utils/results.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from ukbb_pan_ancestry.resources.results import *
-from ukbb_pan_ancestry.heritability.import_heritability import get_h2_ht
+#from ukbb_pan_ancestry.heritability.import_heritability import get_h2_ht
```
Comment (mkanai) [inline utils/results.py]: I think you can delete this comment line now

**Review by mkanai — APPROVED** [review]
Thanks @rahulg603! Overall, looks great to me! 👍

**Comment by rahulg603** [comment]
@mkanai feel free to review the update and then i can merge

**Review by mkanai — APPROVED** [review]
LGTM! 👍

**Pull request merged**